doc/devel/conduct/CoC_reporting_manual.rst

```diff
@@ -0,0 +1,222 @@
+:orphan:
+
+.. _CoC_reporting_manual:
+
+Matplotlib Code of Conduct - How to follow up on a report
+---------------------------------------------------------
+
+This is the manual followed by Matplotlib's Code of Conduct Committee. It's
+used when we respond to an issue to make sure we're consistent and fair.
+
+Enforcing the Code of Conduct impacts our community today and for the future.
+It's an action that we do not take lightly. When reviewing enforcement
+measures, the Code of Conduct Committee will keep the following values and
+guidelines in mind:
+
+* Act in a personal manner rather than impersonal.  The Committee can engage
+  the parties to understand the situation, while respecting the privacy and any
+  necessary confidentiality of reporters.  However, sometimes it is necessary
+  to communicate with one or more individuals directly: the Committee's goal is
+  to improve the health of our community rather than only produce a formal
+  decision.
+
+* Emphasize empathy for individuals rather than judging behavior, avoiding
+  binary labels of "good" and "bad/evil". Overt, clear-cut aggression and
+  harassment exists and we will be address that firmly.  But many scenarios
```

Comment:
There's an extra verb in this sentence, not sure which one of 'be' or 'address' should be changed/removed.